src/borda_clj/client.clj

```diff
@@ -0,0 +1,60 @@
+(ns borda-clj.client
+  (:use [slingshot.slingshot :only [throw+ try+]])
+  (:require [aleph.http :as http]
+            [byte-streams :as bs]
+            [cheshire.core :as json]))
+
+(defn collect
+  "Adds the given measurement (values and dimensions) to the given hashmap and caps the size of the hashmap at the given max-buffer-size"
+  [measurements max-buffer-size dimensions values]
+  (let [key (merge (sorted-map) dimensions)]
```

Comment:
`into` is more idiomatic for this but it doesn't really matter.

Reply:
Will do!